pkg/autodiscovery/providers/ecs.go

```diff
@@ -43,7 +43,13 @@ func (p *ECSConfigProvider) IsUpToDate() (bool, error) {
 // Collect finds all running containers in the agent's task, reads their labels
 // and extract configuration templates from them for auto discovery.
 func (p *ECSConfigProvider) Collect() ([]integration.Config, error) {
-	meta, err := ecsmeta.V2().GetTask()
+	client, err := ecsmeta.V2()
+	if err != nil {
+		log.Debugf("error while initializing ECS metadata V2 client: %s", err)
```

Comment:
we don't need to log here, the error is logged already https://github.com/DataDog/datadog-agent/blob/master/pkg/autodiscovery/autoconfig.go#L203

Reply:
I see, so should I just do `client, _`?

Reply:
no we should indeed return the error, but without logging this `log.Debugf("error while initializing ECS metadata V2 client: %s", err)`

Reply:
Will do 👍